client/my-sites/menus/main.jsx

```diff
@@ -225,7 +225,7 @@ var Menus = React.createClass( {
 					section="menus"
 					secondaryAction={ this.translate( 'Open Classic Menu Editor' ) }
 					secondaryActionURL={ site.options.admin_url + 'nav-menus.php' }
-					secondaryActionTarget="_blank"
+					secondaryActiontarget="_blank" rel="noopener noreferrer"
```

Comment:
- This doesn't do anything. Followed the logic through to `components/empty-content` where `target` is derived from the `secondaryActionTarget`, but the `rel` won't be applied. We may want to enhance `<EmptyContent />` to apply the `rel` when `target` is specified.
- For consistency, the added prop should be on its own line